api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCcrCohortQueries.java

```diff
@@ -0,0 +1,3014 @@
+                    tbMetadata.getIndeterminate().getConceptId())),
+            mapping));
+
+    cd.addSearch(
+        "hivExposure",
+        map(
+            getChildrenWithVisitReason(
+                Collections.singletonList(
+                    commonMetadata.getRecenNascidoMaeHivPositivoConcept().getConceptId())),
+            mapping));
+
+    cd.setCompositionString("(firstConsultation AND rapidTest) AND NOT hivExposure");
+    return cd;
+  }
+
+  /**
+   * CCR-FR22
+   *
+   * <p><b>Indicador 16 - </b> Crianças não expostas ao HIV, testadas com Teste Rápido que tiveram
+   * resultado positivo
+   *
+   * <p>O sistema irá produzir o Indicador 16 “Total de crianças não expostas ao HIV, testadas com
+   * Teste Rápido que tiveram resultado positivo”, da seguinte forma:
+   *
+   * <ul>
+   *   <li>Incluindo todas as crianças que tiveram a 1ª consulta durante o período de reporte (CCR-
+   *       FR7)
+   *   <li>Filtrando as que tiveram o registo de “HIV (teste rápido)” igual a "Positivo” na primeira
+   *       “Ficha de Seguimento de CCR” registada durante o período de reporte (“Data da Consulta”
+   *       >= “Data Início” e <= “Data Fim”).
+   *   <li>Excluindo todas crianças com exposição ao HIV durante o período de reporte (CCR-FR11)
+   * </ul>
+   *
+   * Nota: em caso de existirem mais que uma “Ficha de Seguimento de CCR” durante o período será
+   * considerada a informação registada na primeira ficha.
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition getChildrenNotExposedToHivWithPositiveTestResult() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.setName(
+        "Crianças não expostas ao HIV, testadas com Teste Rápido que tiveram resultado positivo");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Health Facility", Location.class));
+
+    cd.addSearch("firstConsultation", map(getPatients1stConsultation(), mapping));
+
+    cd.addSearch(
+        "rapidTestPositive",
+        map(
+            getGeneralCcrQuery(
+                hivMetadata.getHivRapidTest1QualitativeConcept(),
+                Collections.singletonList(hivMetadata.getPositive().getConceptId())),
+            mapping));
+
+    cd.addSearch(
+        "hivExposure",
+        map(
+            getChildrenWithVisitReason(
+                Collections.singletonList(
+                    commonMetadata.getRecenNascidoMaeHivPositivoConcept().getConceptId())),
+            mapping));
+
+    cd.setCompositionString("(firstConsultation AND rapidTestPositive) AND NOT hivExposure");
+    return cd;
+  }
+
+  public CohortDefinition getChildrenWhoCompletedsoziazida() {
+    SqlCohortDefinition cd = new SqlCohortDefinition();
+    cd.setName("Crianças que completaram Isonizada – coorte de 9 meses");
```

Comment:
javadocs and make sure the method name matches the description name

Reply:
Resolved